drivers/watchdog/wdt_nrfx.c

```diff
@@ -15,23 +15,34 @@
 
 DEVICE_DECLARE(wdt_nrfx);
 
+/* Macro for calculating watchdog ticks from time in milliseconds.
+ */
+#define WDT_NRF_MS_TO_TICKS(time_ms) ((time_ms * 32768) / 1000)
+
+/* Macro for calculating time in milliseconds from watchdog ticks.
+ */
+#define WDT_NRF_TICKS_TO_MS(ticks) ((ticks * 1000) / 32768)
```

Comment:
Typecast `ticks` to `(u32_t)` or use `1000uL` to make sure the multiplication is done on unsigned long integers. Similar story with `time_ms` above.

Reply:
I'll move it to code